app/controllers/timeline_entries_controller.rb

```diff
@@ -31,6 +31,18 @@ def update
     end
   end
 
+  def destroy
+    @timeline_entry = coronavirus_page.timeline_entries.find(params[:id])
+
+    message = if @timeline_entry.destroy && draft_updater.send
+                { notice: "Timeline entry was successfully deleted." }
+              else
+                { alert: "Timeline entry couldn't be deleted" }
```

Comment:
Just wondered if these should be in the locale file? it's not so much a suggestion, just wondering what consistently is and isn't in it.

Reply:
They should. I'll move them in separate commit. There needs to be another review of the content as I think some of it was added at the same time as locale file work.